packages/client/src/components/Modals/AddAgency.vue

```diff
@@ -47,7 +47,8 @@
           invalid-feedback="Required"
         >
           <template slot="label">Code</template>
-          <template slot="description">This should match the Agency Code field in ARPA Reporter spreadsheet uploads. If not using ARPA Reporter, you can set this the same as Abbreviation. This field must be unique across agencies.</template>
+          <template slot="description">This should match the Agency Code field in ARPA Reporter workbook uploads. If not using
+            ARPA Reporter, you can set this the same as Abbreviation. This field must be unique across agencies.</template>
```

Comment:
Were these line breaks (here and below) intentional, or were they added by your IDE preferences?

Reply:
Good catch! I chased this down to a conflict between prettier and eslint, which was caused by the `formatOnSave` option. This is underspecified for which formatter to run! 

More on this [here](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint):

> Please also note that if you use ESLint as your default formatter you should turn off editor.formatOnSave when you have turned on editor.codeActionsOnSave. Otherwise you file gets fixed twice which in unnecessary.

This setting was suggested by the original readme. I've since updated the readme to defer to an actual settings section of the workspace file and added these settings to the workspace with my latest change to ensure greater consistency / remove a docs<->settings discrepancy